react/features/welcome/components/WelcomePage.native.js

```diff
@@ -43,6 +52,23 @@ class WelcomePage extends AbstractWelcomePage {
      */
     static propTypes = AbstractWelcomePage.propTypes;
 
+    /**
+     * Constructor of the Component.
+     *
+     * @inheritdoc
+     */
+    constructor(props) {
+        super(props);
+
+        this.state.hintBoxAnimation = new Animated.Value(0);
+
+        this._getHintBoxStyle = this._getHintBoxStyle.bind(this);
```

Comment:
I don't think _getHintBoxStyle needs binding.

Reply:
Indeed! Also part of the welcome screen PR, so I'll put in another one